src/js/big-play-button.js

```diff
@@ -8,16 +11,20 @@
  * @class
  * @constructor
  */
-vjs.BigPlayButton = vjs.Button.extend();
+var BigPlayButton = Button.extend();
```

Comment:
I actually tried switching to eslint but I think I ran into issues with es6 modules. If you want to see if you can get past that that'd be cool.